### DISCUSSION TITLE
Github Profile 實用小工具

### DISCUSSION THREAD

**Discussion by awkward-willy** [1101-GS4719-C/Python-Community-GS4719-C#636] [優秀專案]
Github 身為全球最大的男性交友平台，勢必得讓人擁有美美的自我介紹說明頁
先前老師在 hw0 的 git 作業中也有請大家操作
但只是文字的內容怎麼能吸引到目光呢，勢必得配上許多酷酷的圖表

有許多專案可以協助生成如 Top Used Languages 之類的圖表
如：
![](http://github-profile-summary-cards.vercel.app/api/cards/repos-per-language?username=vn7n24fzkq&theme=default)

舉例來說：
1. 
專案名稱：github-profile-summary-cards
專案網址：https://github.com/vn7n24fzkq/github-profile-summary-cards.git
專案網站：https://github-profile-summary-cards.vercel.app/demo.html
2.
GitHub Readme Stats
專案名稱：github-readme-stats
專案網址：https://github.com/anuraghazra/github-readme-stats

這裡也放上許多神人的 github profile，大家可以進去欣賞欣賞
也祝大家在 Github上交友順利？
https://github.com/WaylonWalker
https://github.com/donavon
https://github.com/CyrisXD

資料參考：[網址](https://medium.com/starbugs/%E5%A6%82%E4%BD%95%E5%BB%BA%E7%AB%8B%E7%8D%A8%E4%B8%80%E7%84%A1%E4%BA%8C%E7%9A%84-github-profile-%E8%88%87%E4%B8%89%E5%80%8B%E5%BE%88%E9%85%B7%E7%9A%84%E8%A8%AD%E8%A8%88%E5%8F%8A%E6%87%89%E7%94%A8-ef1cbb4b42c1)

**Comment by jason79461385** [comment]
還可以用貪吃蛇的方式來顯示出個人活躍狀態 可以看我個人專業 還蠻有趣的

**Comment by 1chooo** [comment]
確實挺實用的，大家有需要也可以點進我的 profile 參考，有一個精美的 GitHub Porfile 不只自己看了高興，也可以讓他人快速認識我們！

**Reply by chia-kai-chang** [reply]
助教大大的Github太精彩了~ 頭像也超帥~

**Reply by 1chooo** [reply]
哈！謝謝老師的讚美！持續努力中！

**Comment by jylin34** [comment]
我覺得這個東西十分實用，我也在我的profile新增了這個工具，讓我的個人首頁變得十分精美，感謝版主的分享! 超讚 
我也看過其他人有製作[顯示leetcode status](https://github.com/JacobLinCool/LeetCode-Stats-Card)的工具，也很有趣!

**Comment by benbebobo149** [comment]
看到這一篇專案，我就想到說一個工程師或是一個人工作上成就高或底，不僅僅取決於他工作的能力，而是如何清楚且有條理地向主管表達你的貢獻。這篇文章提供了一個很好的工具，讓我意識到打造出一個有個性的profile在工作上是非常重要的。透過清晰地展示我的技能、專業知識和成就，我可以讓主管更清楚地了解我的價值和貢獻，進而提升在團隊中的地位和影響力。這樣的個人品牌建構不僅對當下的工作有益，也將在未來的職涯發展中帶來巨大的優勢。感謝同學的分享，讓我意識到了這一重要的觀點。
